src/snowflake/connector/aio/auth/_idtoken.py

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+#
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+from ...auth.idtoken import AuthByIdToken as AuthByIdTokenSync
+from ._by_plugin import AuthByPlugin
+from ._webbrowser import AuthByWebBrowser
+
+if TYPE_CHECKING:
+    from .._connection import SnowflakeConnection
+
+
+class AuthByIdToken(AuthByPlugin, AuthByIdTokenSync):
```

Comment:
same here, lets name it `AuthByPluginAsync`?

Reply:
updated